### PR TITLE
SG-21170 [ZD to AKN] Update links referenced from code to the old support site to point to AKN

### PR DESCRIPTION
**Pull request by 000paradox000**
This PR replaces ZD urls to AKN and developer urls

### CODE REVIEW

File: python/ui/setup_project.py [hunk 1]
```diff
@@ -650,8 +650,8 @@ def retranslateUi(self, Wizard):
...
-        self.local_file_link_lbl.setText(QtGui.QApplication.translate("Wizard", "Linking to local files must be enabled in your <a href=\"https://support.shotgunsoftware.com/hc/en-us/articles/219030938\">ShotGrid Preferences</a>", None, QtGui.QApplication.UnicodeUTF8))
+        self.storage_def_lbl.setText(QtGui.QApplication.translate("Wizard", "This configuration requires one or more filesystem roots where files will be stored. For each filesystem root required by this configuration on the left, specify an associated ShotGrid Local Storage on the right that defines the paths for the operating systems you use. <a href=\"https://help.autodesk.com/view/SGSUB/ENU/?guid=SG_Administrator_ar_data_management_ar_linking_local_files_html_to_set_up\">More info…</a>", None, QtGui.QApplication.UnicodeUTF8))
```
Comment (mathurf) [inline python/ui/setup_project.py:653]: That URL bring me to a 404 page :/

**Review by mathurf — CHANGES_REQUESTED** [review]
Need to fix one of the URL. It is pointing to a 404 page

File: python/ui/setup_project.py (hunk 1 above)
Comment (000paradox000) [reply python/ui/setup_project.py]: Done

File: python/ui/setup_project.py [hunk 2]
```diff
@@ -650,8 +650,8 @@ def retranslateUi(self, Wizard):
...
-        self.local_file_link_lbl.setText(QtGui.QApplication.translate("Wizard", "Linking to local files must be enabled in your <a href=\"https://support.shotgunsoftware.com/hc/en-us/articles/219030938\">ShotGrid Preferences</a>", None, QtGui.QApplication.UnicodeUTF8))
+        self.storage_def_lbl.setText(QtGui.QApplication.translate("Wizard", "This configuration requires one or more filesystem roots where files will be stored. For each filesystem root required by this configuration on the left, specify an associated ShotGrid Local Storage on the right that defines the paths for the operating systems you use. <a href=\"https://help.autodesk.com/view/SGSUB/ENU/?guid#how-do-i-set-up-local-file-storage-for-networked-resources-such-as-a-shared-element-library-\">More info…</a>", None, QtGui.QApplication.UnicodeUTF8))
```
Comment (mathurf) [inline python/ui/setup_project.py:653]: I think https://help.autodesk.com/view/SGSUB/ENU/?guid#how-do-i-set-up-local-file-storage-for-networked-resources-such-as-a-shared-element-library-\
should be https://help.autodesk.com/view/SGSUB/ENU/?guid=SG_Administrator_ar_data_management_ar_linking_local_files_html

**Review by mathurf — CHANGES_REQUESTED** [review]

**Review by mathurf — APPROVED** [review]
LGTM!

**Pull request merged**